AdyenNetworking/APIClient/APIClient.swift

```diff
@@ -18,6 +18,9 @@ public protocol APIClientProtocol: AnyObject {
     /// Performs the API request.
     func perform<R: Request>(_ request: R, completionHandler: @escaping CompletionHandler<R.ResponseType>)
     
+    /// :nodoc:
+    /// Performs the API download request.
+    func performDownload<R: Request>(_ request: R, completionHandler: @escaping CompletionHandler<R.ResponseType>) where R.ResponseType == DownloadResponse
```

Comment:
really Nice! A+.

small thing though, I suggest to change the naming a bit:

```Swift
func perform<R: Request>(downloadRequest request: R, completionHandler: @escaping CompletionHandler<R.ResponseType>) where R.ResponseType == DownloadResponse
```

this is a bit more swifty :-)